pywbem_mock/_mainprovider.py

```diff
@@ -3146,16 +3160,28 @@ def OpenQueryInstances(self, namespace, FilterQueryLanguage, FilterQuery,
         self._validate_open_params(FilterQueryLanguage, FilterQuery,
                                    OperationTimeout)
 
-        # pylint: disable=assignment-from-no-return
         # Issue #2064 TODO/ks implement execquery
-        # Handle ReturnQueryResultClass maybe or make it exception
+        # pylint: disable=assignment-from-no-return
         instances = self.ExecQuery(namespace, FilterQueryLanguage, FilterQuery)
 
-        return self._open_response(namespace, instances,
-                                   'PullInstancesWithPath',
-                                   OperationTimeout,
-                                   MaxObjectCount,
-                                   ContinueOnError)
+        if ReturnQueryResultClass:
```

Comment:
I think this is correct code but not sure why it became issue now since the ExecQuery always returns not implemented.

Reply:
This is one of the errors you get when running the pywbemtools test against pywbem 1.0.0.